packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx

```diff
@@ -137,6 +151,27 @@ export function CreateForm(props: CreateFormProps) {
         }
     }, [])
 
+    // qualification
+    const [qualification, setQualification] = useState('')
+    const [verify, setVerify] = useState(false)
+    const qualificationChange = useCallback(async (ev: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Better create a hook for this. Two reasons:

+ For reusing the logic as a hook instead of hardcoding in UI.
+ The current implementation is buggy. It will run into race condition in some situations.

If you need a hook example. There are many in the `web3/hooks` folder.

Reply:
done